server/weaverbird/steps/aggregate.py

```diff
@@ -8,9 +8,11 @@
 from weaverbird.steps.base import BaseStep
 from weaverbird.types import ColumnName, PopulatedWithFieldnames, TemplatedVariable
 
-AggregateFn = Literal['avg', 'sum', 'min', 'max', 'count', 'count distinct', 'first', 'last']
+AggregateFn = Literal[
+    'avg', 'sum', 'min', 'max', 'count', 'count distinct', 'first', 'last', 'count_even_null'
+]
 
-functions_aliases = {'avg': 'mean', 'count distinct': 'nunique'}
+functions_aliases = {'avg': 'mean', 'count distinct': 'nunique', 'count_even_null': len}
```

Comment:
Is there a difference between using `len` and using `nuunique(drop_na=False)`?
https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.nunique.html

Reply:
I think it returns the same result, however, using len allow us to have a simpler code.

If we go this way, I think it would be cleaner to make it its own step than to modify the  aggregate step for this very specific behavior

Reply:
Good, thanks for the answer